source/documentation/adrs/adr-006.html.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: ADR-006 Migrate to Github Actions instead of CircleCI
+last_reviewed_on: 2022-02-09
+review_in: 12 months
+---
+
+# <%= current_page.data.title %>
+
+### Date: 14-02-2023
+
+## Status
+
+✅ Accepted
+
+## Context
+
+CircleCI had a [serious security breach](https://circleci.com/blog/jan-4-2023-incident-report/) exposing all environment variables and tokens within their platform.
+
+CircleCI exposes all environment variables within a workflow to every step in the workflow. This was also exploited historically with a [Codecov](https://about.codecov.io/security-update/) exploit to exfiltrate environment variables. 
+
+CircleCI has no differential between standard environment variables (such as compile flags, build targets etc) and what should be privileged information.
+
+Github Actions provides first class support for passwords, masking them for output and redacting them from logs. Access permissions to environment variables can be controlled per step, reducing data leakage to third parties. Actions can have access permissions scoped, reducing what features they can use.
+
+## Decision
+
+All future projects to utilise Github Actions for pipelines.
```

Comment:
I suppose we should also add a nod to the fact that Sirius will be on Jenkins until we can reduce the build overhead.

Reply:
good point - will put that in

Reply:
adjusted to have a sirius caveat